v4/export/sql.go

```diff
@@ -427,7 +426,13 @@ func resetDBWithSessionParams(db *sql.DB, dsn string, params map[string]string)
 	}
 
 	for k, v := range support {
-		dsn += fmt.Sprintf("&%s=%s", k, url.QueryEscape(v))
+		var s string
+		if str, ok := v.(string); ok {
+			s = wrapQuotation(str)
```

Comment:
let's just wrap the quotation mark directly. the only case the `str` passes the `strings.HasPrefix` stuff is when user invokes dumpling as

```sh
./dumpling --snapshot "'stuff'"
```

which is so unnatural that there's no point supporting it.

```suggestion
			s = wrapStringWith(str, "'")
```

(the `wrapBackTicks` function probably has the same bug, let's check it in the future)

Reply:
ok